src/chi/utilities/anchor.scss

```diff
@@ -0,0 +1,51 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+
+  .-anchor {
+    &::before {
+      content: '';
+      display: block;
+      height: 5rem;
+      margin: -5rem 0 0;
+    }
+
+    &:hover {
+      & span {
+        & a {
+          color: $brand-color;
+          cursor: pointer;
+          font-weight: 600;
+          opacity: 1;
+          text-decoration: none;
+          transition: opacity 0.15s ease-in-out;
+        }
+      }
+    }
+
+    & span {
+      & a {
+        color: $brand-color;
+        cursor: pointer;
+        font-weight: 600;
+        opacity: 0;
+        transition: opacity 0.15s ease-in-out;
+
+        &:hover {
+          opacity: 1;
+          text-decoration: none;
+        }
+      }
+    }
+
+    @media only screen and (max-width: $md-breakpoint) {
```

Comment:
@davidojedacl what is purpose of this media query?

Reply:
In small screens the header is not fixed . So not is necessary to put 5rem marginl of distance because the header not cover the title
<img width="1269" alt="screen shot 2019-02-19 at 4 30 51 pm" src="https://user-images.githubusercontent.com/43439742/53026889-22661480-3464-11e9-9f6d-887570b371b7.png">

Reply:
@davidojedacl these properties must be in chi-docs.scss and can not be as part of the general anchor.scss, as we don't want to make it generic